src/annotator/sidebar.js

```diff
@@ -183,8 +174,11 @@ export default class Sidebar extends Guest {
   destroy() {
     this._unregisterEvents();
     this._hammerManager?.destroy();
-    this.frame?.remove();
-    this.hypothesisSidebar?.remove();
+    if (this.hypothesisSidebar) {
+      this.hypothesisSidebar.remove();
```

Comment:
To check my understanding - this is because if `this.hypothesisSidebar` exists, we assume it contains `this.iframe` and so that no longer needs to be removed?

Reply:
Correct. `this.iframe` element is a children of the hypothesisSidebar's shadow DOM.

A new addition is that now we remove `this.iframe` from the `externalContainer` on `destroy`. That's was something overlooked previously. That's the reason for the extra test.